feat(dynamite): introduce dynamite config to set ignore directives (#1025)
nextcloud/neon: packages/dynamite/dynamite/README.md

> Provides [Dart Build System](https://github.com/dart-lang/build) builders for generating clients from an  [OpenAPI specifications](https://swagger.io/specification/).

Reviewer: ```suggestion
Provides a [Dart Build System](https://github.com/dart-lang/build) builder for generating clients from an [OpenAPI specifications](https://swagger.io/specification/).
```

> The builders generate code when they find files with an `.openapi.json` or `.openapi.yaml` extension in the lib directory.

Reviewer: ```suggestion
The builder generates code if it find files with an `.openapi.json` or `.openapi.yaml` extension in the lib directory.
```

# Build configuration

> Aside from setting arguments on the associated annotation classes, you can also

Reviewer: We don't provide any annotations?
